operated/shifting_spatial_id_test.go

```diff
@@ -239,6 +254,51 @@ func TestGetShiftingSpatialID07(t *testing.T) {
+
+	nLayer := 5
+
+	start := time.Now()
+	resultVal, error := GetNspatialIdsAroundVoxcels(ids, int64(nLayer), int64(nLayer))
+	end := time.Since(start)
+	if error != nil {
+		t.Fatal(error)
+	}
+
+	fmt.Println(len(resultVal))

```

Comment:
Thank you for your test functions, but they have two problems.
The other test functions compare the values with the expected values.
Please check the values, too.
And the functions of fmt always print the messages even if the test is succeeded.
The other test functions use testing.T.Log(\_) e.t.c.
They prints the messages if the tests are failed, so we can find the failures easily.
Please use testing.T.Log(\_).
The same applies the others.